cmd/devstream/init.go

```diff
@@ -30,3 +43,35 @@ func initCMDFunc(cmd *cobra.Command, args []string) {
 
 	log.Success("Initialize finished.")
 }
+
+// compareDtmMD5 compare dtm itself and remote md5, if does not match, cannot  download plugins from remote
```

Comment:
Don't put a lot of code in the application directory. If you think the code can be imported and used in other projects, then it should live in the /pkg directory. If the code is not reusable or if you don't want others to reuse it, put that code in the /internal directory. Read our [Project Layout](https://github.com/merico-dev/stream/blob/main/docs/project_layout.md#cmd) doc again.

Reply:
yes，I will move them into dir /pkg/md5